x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -1484,4 +1484,48 @@ export const tasks: TelemetryTask[] = [
       };
     },
   },
+  {
+    name: 'top_traces',
+    executor: async ({ indices, telemetryClient }) => {
+      const response = await telemetryClient.search({
+        index: indices.transaction,
+        body: {
+          size: 0,
+          track_total_hits: false,
+          timeout,
+          query: {
+            bool: {
+              filter: [range1d],
+            },
+          },
+          aggs: {
+            top_traces: {
+              terms: {
+                field: 'trace.id',
```

Comment:
nit: use the constant

```suggestion
                field: TRACE_ID,
```